packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -1059,15 +1092,19 @@ export default class SalesforceCRMService implements CRM {
     recordId,
   }: {
     existingFields: Field[];
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I think this is a good place to use `any` here because these are all custom fields that are unique to each Salesforce instance.